staging/src/k8s.io/code-generator/cmd/conversion-gen/main.go

```diff
@@ -52,9 +52,12 @@ func main() {
 	arguments.GoHeaderFilePath = filepath.Join(args.DefaultSourceTree(), "k8s.io/kubernetes/hack/boilerplate/boilerplate.go.txt")
 
 	// Custom args.
+	// TODO: make callers pass this in.  It is too opaque here, and any use of
+	// the flag that DOESN'T include these is broken.
 	customArgs := &generators.CustomArgs{
 		ExtraPeerDirs: []string{
 			"k8s.io/kubernetes/pkg/api",
+			"k8s.io/kubernetes/pkg/api/v1",
```

Comment:
what breaks without this line?

Reply:
If you have a case where you're rebuilding a small number of packages and none of them declare this as a conversion peer, it fails to find the PodTemplate conversion function and replaces it with the default "inefficient conversion" message.